openwisp_controller/config/base/vpn.py

```diff
@@ -72,7 +84,9 @@ def save(self, *args, **kwargs):
         if not self.cert:
             self.cert = self._auto_create_cert()
         if not self.dh:
-            self.dh = self.dhparam(2048)
+            self.dh = self._placeholder_dh
+            with transaction.atomic():
```

Comment:
after some investigation I concluded we don't have to use atomic in this case, we would have to use it if we would be executing write queries (so they all get rolledback if there's an error)